compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/bufferize_to_allocation_pack.mlir

```diff
@@ -10,17 +10,17 @@ func.func @matmul_static(%arg0 : tensor<16x256xi8>, %arg1 : tensor<256x256xi8>)
   %5 = tensor.empty() : tensor<16x256xi32>
   %6 = linalg.fill ins(%c0_i32 : i32) outs(%5 : tensor<16x256xi32>) -> tensor<16x256xi32>
   %7 = tensor.empty() : tensor<1x4x16x64xi8>
-  // CHECK-1: memref.alloc() : memref<1x4x16x64xi8, 1>
-  // CHECK-1: bufferization.to_tensor %{{.*}} restrict writable : memref<1x4x16x64xi8, 1>
+  // CHECK-1: memref.alloc() : memref<1x4x16x64xi8, 1 : i32>
```

Comment:
Is there a particular reason to set this as integer? Is this something modified and used in AIR?

Reply:
by default this was using an `IntegerAttr` with `IndexType`. Now its an `I32IntegerAttr` since we are using enums.